### PR TITLE
Feature/travis ci support

### LINKED ISSUE #297: documentation: replace references to old image names
In b0b5ad3312cd0dbd1a7da368664294f1ad6cd81c the explicit `osmaxx` prefix has been dropped. Some parts of the checked-in documentation still mention image names with the prefix, though:
```bash
git grep -E 'osmaxx[a-z]+'
```
```
docs/project-development-environment.md:docker-compose run osmaxxwebappdev /bin/bash
docs/project-development-environment.md:`docker-compose build --no-cache osmaxxwebappdev`.
docs/project-development-environment.md:docker save osmaxx_osmaxxdatabase > /tmp/osmaxx-database-alpha1.docker-img.tar
docs/project-development-environment.md:docker save osmaxx_osmaxxwebapp > /tmp/osmaxx-webapp-alpha1.docker-img.tar
```

### PR DESCRIPTION
**Pull request by hixi**
*Warning*: Moved the directory structure!

Reviewed by:
- [x] @das-g 
- [ ] @wasabideveloper 

This pull request should already be automatically testes through travis-ci :-)

Accidentally closes #297

### CODE REVIEW

File: compose-development.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-shareddatadev:
+shareddata:
```
Comment (wasabideveloper) [inline compose-development.yml]: Why do we rename all the development containers from -dev to -''?

Comment (hixi) [inline compose-development.yml]: because we don't need that duplication of names, since the compose-files are only applicatble for development.

Comment (wasabideveloper) [inline compose-development.yml]: :heavy_check_mark:

**Comment by hixi** [comment]
@wasabideveloper if you're done (and happy) with the PR, please merge it.

**Pull request merged**